File: C++/021.cpp
// C++ program: Smallest integer

#include<iostream>
using namespace std;
int main()
{
	int l,m,n;
	cout<<"Enter the value of l = ";
	cin>>l;
	cout<<"Enter the value of m = ";
	cin>>m;
	cout<<"Enter the value of n = ";
	cin>>n;

	if(l<m && l<n)
	{
		cout<<l<<" is the smallest integer";
	}
	else if(m<l && m<n)
	{
		cout<<m<<" is the smallest integer";
	}
	else
	{
		cout<<n<<" is the smallest integer";
	}
}

File: Java/021.java
// Java class: Even or odd or zero

package projectFolder;

import java.util.Scanner;

public class EvenOddZero
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);

        System.out.print("Enter any integer = ");
        int num = scan.nextInt();

        if (num == 0)
        {
            System.out.println("The integer is zero");
        }
        else if (num % 2 == 0)
        {
            System.out.println("The integer is even");
        }
        else
        {
            System.out.println("The integer is odd");
        }
    }
}